scripts/lint.sh

```diff
@@ -36,17 +36,16 @@ function find_go_files {
 }
```

Comment:
Should we be removing this function? (or using it?)

Reply:
removed